modules/base.nix

```
@@ -28,5 +28,9 @@
   security.sudo.wheelNeedsPassword = false;
   programs."${userShell}".enable = true;
 
+  system.userActivationScripts.setupShell = pkgs.lib.optionalString (userShell == "zsh") ''
```

How about something like
```suggestion
  # suppress noisy startup menu when ~/.zshrc is missing
  system.userActivationScripts.setupShell = pkgs.lib.optionalString (userShell == "zsh") ''
```

---

How do you think about change requests like this? I would be fine if we commit suggestions like this immediately, but we can also go the traditional route and review them so each author can finish his complete pull reqeust.

What do you think?

---

In general I prefer have reviewers make suggestions and leave the responsibility for the PR and the commits to the PR author.